### DISCUSSION TITLE
`vm` commands not found.

### DISCUSSION THREAD

**Discussion by megabyte0x** [Cyfrin/foundry-full-course-cu#1337] [Q&A] [answered]
I am getting `vm` error in one of my projects. 

Any solution for this error:

```bash
Error (9582): Member "startPrank" not found or not visible after argument-dependent lookup in contract Vm.
```

`vm.startBroadcast` is working though. 

you can find the code line throwing error [here](https://github.com/megabyte0x/GHOPay/blob/b9cc64658cd8e4444392889dd99f17481e832ef2/contracts/test/unit/MainVault.t.sol#L60)

**Comment by ghost** [comment, marked as answer]
Usually that error occurs when you're passing the incorrect type to the cheatcode.

The `startBroadcast` and `broadcast` cheatcodes accept either address or private key as arguments, but `startPrank` and `prank` only accept addresses.

**Reply by megabyte0x** [reply]
and that's the CORREECCCTTT ANSWERRRR!!

Thanks a ton!!
